src/extractUnusedExports.js

```diff
@@ -2,18 +2,26 @@ import lodash from 'lodash';
 
 export default function extractUnusedExports(
   exportedNames,
-  importedIdentifiers
+  importedIdentifiers,
+  importedIdentifiersTest
 ) {
+  const identifierIsNotATest = importedIdentifier =>
+    !importedIdentifiersTest
+      .map(identifier => identifier.sourcePath)
+      .includes(importedIdentifier.sourcePath);
```

Comment:
Looks like here it would be more efficient to build a Set of test identifiers paths and then only check if that path exists.

`
const testPaths = new Set(importedIdentifiersTest.map(identifier => identifier.sourcePath));
const identifierIsNotATest = importedIdentifier => !testPaths.has(importedIdentifier.sourcePath);
`